app/templates/components/donations/donation-progress.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="information">
+  <div class="amount">
+    <div class="value">{{format-currency amountDonated trimZero=true}}</div>
+    <div class="descriptive-text">given each month</div>
+  </div>
+  <div class="percentage">
+    <div class="value">{{format-percentage percentage trimZero=true}}</div>
+    <div class="descriptive-text">of goal</div>
+  </div>
+</div>
+{{progress-bar-container percentage=percentage}}
+<div class="description">{{donationGoal.description}}</div>
```

Comment:
Thoughts on following something like BEM for styling where child elements clearly adopt from their parents?  So, rather than a general `description` tag, we could have its class name based on its wrapper?

```
<div class="donation-goal">
  <h1 class="donation-goal__header">Lorem Ipsum</h1>
  <div class="donation-goal__description">some description</div>
</div>
```
Then in our sass styles, we can follow the convention of:

```
.donation-goal {
  color: whatever;

  &__header {
    font-size: 50px
  }

  &__description {
     font-family: some-font, sans-serif
  }
}
```

It adds explicitness to the styles and clarity into where elements are used.